### PR TITLE
Added 'F' character in straight.h

### LINKED ISSUE #258: Add character: <straight> to font: <F>
*Character* : <straight>
*Font* : <F>

### PR DESCRIPTION
**Pull request by Stunner33**
Hey, i am new to opensource I have add F character in Straight font. If there is any issue plz Let me know.i am ready to contribute

closes #258

### CODE REVIEW

**Comment by AnshumanMahato** [comment]
@Stunner33, thanks for contributing to our project. Although this issue was assigned to someone else, I am merging your PR. But, for further contributions, please get an issue assigned to yourself before opening a PR. This helps in avoiding confusion. I hope you understand, 😊

**Comment by Stunner33** [comment]
@AnshumanMahato I am sorry about that, as i said i am new to open-source so wasn't aware of it...Plz Add hacktoberfest-accepted tag.Thank u🙂

**Comment by AnshumanMahato** [comment]
No worries @Stunner33, Just make the requested changes and ping me here. I'll check and merge.

**Comment by Stunner33** [comment]
@AnshumanMahato Ok.I have made the changes for about issues and sent pr already so plz check it..if there any correction then let me know

**Comment by AnshumanMahato** [comment]
@Stunner33 did you commit those changes?

**Comment by Stunner33** [comment]
> @Stunner33 did you commit those changes?
> 

I did make the changes, can you please check it once from your end?

**Comment by AnshumanMahato** [comment]
> > @Stunner33 did you commit those changes?
> 
> I did make the changes, can you please check it once from your end?

Well, all I have here is that you added the letter F. There is nothing for the changes that I requested after that. You can see the above messages. I have mentioned some changes that I want here.

**Comment by Stunner33** [comment]
> > > @Stunner33 did you commit those changes?
> > 
> > I did make the changes, can you please check it once from your end?
> 
> Well, all I have here is that you added the letter F. There is nothing for the changes that I requested after that. You can see the above messages. I have mentioned some changes that I want here.

Plz can u re-mention the changes as i can't find ur message for changes

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/Straight/straight.h [hunk 1]
```diff
@@ -72,16 +72,19 @@ class Straight : public Fonts
...
-    /*
+    
         char **F()
         {
             char **character = getCharGrid();
 
             //Enter the character grid in ROWS X COLS
+            character[1][0]=character[2][0]='|';
+            character[0][1]=character[0][2]=character [0][3]=character[1][1]=character[1][2]='_';
+            character [][]=' ';
```
[the remark this answers is not in the document]
Comment (AnshumanMahato) [reply Fonts/Straight/straight.h:83]: line no 83

**Review by AnshumanMahato — APPROVED** [review]

**Comment by AnshumanMahato** [comment]
everything is fine... thanks for contributing @Stunner33. 😊 For further contributions, look for missing characters in any fonts. If there is no existing issue open and assigned for that, open an issue for yourself and I will assign that you.

**Comment by Stunner33** [comment]
> everything is fine... thanks for contributing @Stunner33. 😊 For further contributions, look for missing characters in any fonts. If there is no existing issue open and assigned for that, open an issue for yourself and I will assign that you.

Thank u@AnshumanMahato and sorry for trouble caused. I may have disturb u by asking too many questions..My apologies 🙂

**Comment by AnshumanMahato** [comment]
No worries @Stunner33 . Happy to help. That's what we are here for 😊.

**Pull request merged**